test/integration/cli/hookfiles-cli-test.js

```diff
@@ -201,7 +201,7 @@ describe('CLI', () => {
           // TCP server echoing transactions back
           const hookHandler = net.createServer((socket) => {
             socket.on('data', data => socket.write(data));
-            socket.on('error', err => console.error(err));
```

Comment:
I wonder, is this as useful for debugging?

I thing coercing an error to string won't show stacktrace like `console.error` will so this might make it harder to debug tests. But it is late so I didn't dig into this, will proceed with review tomorrow.

Reply:
Hmmm, I didn't think of this behavior of `console.error`.